custom/icds_reports/tasks.py

```diff
@@ -1764,18 +1764,17 @@ def drop_df_indices(agg_date):
             cursor.execute(query)
 
 
-@periodic_task_on_envs(
-    settings.ICDS_ENVS,
-    run_every=crontab(minute=0, hour=14),  # To run on 7:30 PM IST
-    acks_late=True,
-    queue='icds_aggregation_queue'
-)
 def update_governance_dashboard(target_date=None):
     if target_date is None:
         target_date = date.today()
+    current_month = target_date.replace(day=1)
+    _agg_governance_dashboard(current_month)
 
-    month = target_date.replace(day=1)
 
-    db_alias = router.db_for_write(AggGovernanceDashboard)
-    with transaction.atomic(using=db_alias):
-        AggGovernanceDashboard().aggregate(month)
+@task(queue='icds_aggregation_queue')
```

Comment:
Why is this a celery task? if it is already called in a management command it does not need to be async (and its not called asynchronously above anyway)

Reply:
I meant to call is asyn above.  async because this will be called as a part of aggregation at the last. I dont want to block the success email. basically I want this to be same as MBT